en/frames_vtol/vtol_tiltrotor_omp_hobby_zmo_fpv.md

```diff
@@ -0,0 +1,274 @@
+
+The sensor can be mounted on top of the 3D-Printed mount.
+
+![Pitot tube 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/pitot-tube-02.jpg)
+
+#### Lidar
+
+If needed a lidar can be installed in the front of the fuselage. To install the Lidar, remove the heat sink, and glue the lidar together with the 3D-Printed lidar mount into place.
+
+![Lidar 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/lidar-01.jpg)
+
+#### GPS
+
+To mount the GPS, screw the 2 3D-Printed parts with 3x M3x10 screws together. Take the GPS out of the plastic case and unplug the connector. Feed the cable through the carbon spar. Glue the 3D-Printed part with 5 min epoxy in place. After the glue cured, screw the GPS with 4x M2.5x10 screws to the plate.
+
+Glue the GPS mount into place
+
+![GPS 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/gps-01.jpg)
+
+Screw the GPS to the mount
+
+![GPS 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/gps-02.jpg)
+
+#### USB-Camera
+
+Cut the USB cable of the camera so that the length is 15 cm. Cut the USB-Adapter cable to be 25 cm and solder the two cables together. To install the camera you need to cut a hole into the foam of the wall. Then you can mount the camera with double sided tape to the wall.
+
+Hole to feed the USB cable through the wall.
+
+![USB Camera 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/camera-01.jpg)
+
+### Flight controller
+
+The Flight controller can be installed above the ESC. If a Pixhawk 6c or 6c mini is used, simply stick the flight controller with double sided tape into place.
+If a Skynode is used, place it at the on top of the ESCs and mark the 2 rear mounting locations on the injection molded plastic part of the ZMO. Remove the Skynode from the vehicle and drill 2 holes with a 2.8 mm drill bit into the plastic part. Put the Skynode back into place and screw it down with 2x M3x10 screws. An other option is to add some threaded inserts into the holes. Since the injection molded part of the ZMO is very thin, they need to be glued in place. Screw the front Skynode mount with 2x M3x10 screws at the Skynode. Then add some 5 min epoxy at the bottom of the mount and put a weight on top of the Skynode until the glue is cured. To better reach the 2 mounting screws at the front, poke 2 holes from the top through the foam so you can access the screws.
+
+Mounting holes for the Skynode in the back
+
+![Flight controller 01](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-01.jpg)
+
+Skynode mount in the front
+
+![Flight controller 02](../../assets/airframes/vtol/omp_hobby_zmo_fpv/flight-controller-02.jpg)
+
+### Antennas and RC receiver
+
+One LTE antenna can be installed at the bottom of the vehicle. For that you can feed the antenna wire through the opening for the ESC heat-sink. The second antenna can be installed on the inside of the vehicle on the left side of the battery compartment. The RC receiver can also be placed at the left side of the battery compartment.

```

Comment:
Specify that the LTE antennas are from the Skynode LTE module, and that builds with a Pixhawk instead would require some other telemetry link.